src/makeInfinityAdder.js

```diff
@@ -4,7 +4,19 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function add(value) {
+    if(arguments.length === 0){
```

Comment:
Using `arguments.length` is a valid way to check for no arguments, but it's not commonly used in modern JavaScript. An alternative is to check if `value` is `undefined`, which might be more readable to some developers.